.github/workflows/pypi.yml

```diff
@@ -3,35 +3,49 @@ name: deploy
 on:
   push:
     tags:
-      - '*'
+      - "*"
 
 jobs:
-  build-n-publish:
+  publish:
```

Comment:
Why is this different from https://github.com/N3PDF/yadism/blob/master/.github/workflows/pypi.yml ? will this be the new style?

Reply:
I added a cache layer, to skip dependency installation when not needed. I updated even in `yadism`, and I'm going to do it in `banana`.

Actually, this is annoying, since it is the exact same workflow, so I scheduled even to [factorize workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows).
But since this is possible only inside a single organization, if I want to share across repos I have to pick one to put them all, maybe a new one, and similar complications... I postponed and copy-pasted, for the time being.

Reply:
on the same grounds: should we also duplicate https://github.com/N3PDF/yadism/blob/593a012dc7f4666c2c993dbdd37a6a2d55db0455/install.py ?

Reply:
the answer is no - should go to README